lib/zipkin-tracer/rack/zipkin-tracer.rb

```diff
@@ -94,7 +94,8 @@ def synchronize(&block)
       @lock.synchronize do
         yield
       end
-    rescue# Nothing wonky that the tracer does should stop us from using the app!!!
+    rescue StandardError => e # Nothing wonky that the tracer does should stop us from using the app!!!
+      @config.logger.error("Exception #{e.message} while sending Zipkin traces. #{e.backtrace}")
```

Comment:
I think it should be error level.
Any app should have log rotation or other system to avoid the log to fill the disk space. If we can bring it down with this, other many gems/pieces or their app would be able to do so also.